src/models/MSC3089Branch.ts

```diff
@@ -162,9 +162,8 @@ export class MSC3089Branch {
 
         if (!event) throw new Error("Failed to find event");
 
-        // Sometimes the event isn't decrypted for us, so do that. We specifically set `emit: true`
-        // to ensure that the relations system in the sdk will function.
```

Comment:
`emit: true` is the default, so there's no reason to imagine the "relations system" won't work without it.